### DISCUSSION TITLE
Please add a new model

### DISCUSSION THREAD

**Discussion by hakhagmon** [Arkady23/V380-Web#39] [Q&A]
Hello! 
Please add a new model:
Software: AppV380E31_V1.1.5.1_20210715
Firmware: HwV380E31_WF6_PTZ_WIFI_20201218

thx!

**Comment by Arkady23** [comment]
To add a new camera to the table, you need to find out the Hwname.

**Comment by hakhagmon** [comment]
вот что мне прислали из техподдержки
[SPECIAL_V380_AK_3918E-V330_20MB-DMA_V9.9.9.9_2021-09-24.zip](https://github.com/Arkady23/V380-Web/files/8511270/SPECIAL_V380_AK_3918E-V330_20MB-DMA_V9.9.9.9_2021-09-24.zip)

**Reply by Arkady23** [reply]
Там обозначена Hwname V380E4_C3 и универсальная версия номер 9090909. Последний вариант в таблице должен вам подойти.

**Reply by hakhagmon** [reply]
пробовал, не заработало с ней. и на флешку видео не сейчас

**Reply by Arkady23** [reply]
Может на поддержке сами не знают какой у вас Hwname. Может узнать через трафик?
